### DISCUSSION TITLE
Error on creating new Company project template

### DISCUSSION THREAD

**Discussion by SimoneBergonzi** [mia-platform/community#574] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

v13.2.1

### Description

Hi, I'm trying to create e new project template but getting a 400 status code on POST /templates

this is the error response:
```
{
    "statusCode": 400,
    "code": "FST_ERR_VALIDATION",
    "error": "Bad Request",
    "message": "body/deploy must have required property 'projectStructure'"
}
```

this is the payload:

```
{
  "deploy": {
    "strategy": "push",
    "runnerTool": "mlp"
  },
  "name": "*** template",
  "archiveUrl": "https://git.tools.mia-platform.eu/api/v4/projects/***/repository/archive.tar.gz",
  "description": "*** template with default configuration for enhanced workflow projects"
}
```

![wizard](https://github.com/user-attachments/assets/9887a3ad-771d-4633-aa25-024b4961704b)

### Actual Outcome

Getting a bad request error

### Expected Outcome

I'd like to add e new project template to the Company

**Comment by mattia-fumo** [comment, marked as answer]
Hi @SimoneBergonzi,

we've manged to replicate the error, this seems a bug on the client. For the moment, you could perform the request manually and add the `projectStructure` field in `deploy` and set it to `default`.

It will probably be fixed in the next release `v13.3.1`. I'll let you know.

Thanks for the report.
